### PR TITLE
feat: activate linting before publish

### PR DESCRIPTION
**Pull request by doubleface**
+ fixed linting issues

### CODE REVIEW

File: packages/cozy-jobs-cli/src/cozy-authenticate.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+/* eslint no-console: off */
```
Comment (enguerran) [inline packages/cozy-jobs-cli/src/cozy-authenticate.js]: I think we must use the logger

Or put this rule in the global config file

File: packages/cozy-konnector-libs/scripts/visualizer/index.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ const generate = async options => {
 }
 
 process.on('unhandledRejection', x => {
+  // eslint-disable-next-line no-console
```
Comment (enguerran) [inline packages/cozy-konnector-libs/scripts/visualizer/index.js]: don't like this rule for one line

if we use console.log, just let's put the rule in the config file

File: packages/cozy-konnector-libs/src/libs/hydrateAndFilter.js [hunk 3]
```diff
@@ -133,7 +133,7 @@ const hydrateAndFilter = (documents, doctype, options = {}) => {
   }
 
   const defaultShouldSave = () => true
-  const defaultShouldUpdate = existing => false
+  const defaultShouldUpdate = existing => false // eslint-disable-line no-unused-vars
```
Comment (enguerran) [inline packages/cozy-konnector-libs/src/libs/hydrateAndFilter.js]: mmmhhh, thank you lord eslint!
But it seems that the variable should be a function that always returns `false' regardless of the given parameter.

So why do we have a parameter?

File: packages/cozy-konnector-libs/src/libs/linkBankOperations.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ const flatten = require('lodash/flatten')
 const sumBy = require('lodash/sumBy')
 const geco = require('geco')
 const moment = require('moment')
+const cozyClient = require('./cozyclient')
```
Comment (enguerran) [inline packages/cozy-konnector-libs/src/libs/linkBankOperations.js]: I think we could use the variable stored in `global`, don't we?

File: packages/cozy-logger/src/index.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ let level = LOG_LEVEL || 'debug'
 const format = env2formats[env]
 const filters = [filterLevel, filterSecrets]
 
-const filterOut = function(level, type, message, label, namespace) {
+const filterOut = function() {
```
Comment (enguerran) [inline packages/cozy-logger/src/index.js]: That's a change! 5 useless parameters removed! ;-)

File: packages/cozy-jobs-cli/src/cozy-authenticate.js (hunk 1 above)
Comment (doubleface) [reply packages/cozy-jobs-cli/src/cozy-authenticate.js]: In this case, console.log is used to display a message to the user of the cozy-run-dev cli. I think this is justified.

File: packages/cozy-konnector-libs/scripts/visualizer/index.js (hunk 2 above)
Comment (doubleface) [reply packages/cozy-konnector-libs/scripts/visualizer/index.js]: I actually want to enforce this rule in cozy-konnector-libs. But this file is a script to generate a visualization of links between doctypes and is not run by the stack, then we don't need to use log here.

File: packages/cozy-konnector-libs/src/libs/hydrateAndFilter.js (hunk 3 above)
Comment (doubleface) [reply packages/cozy-konnector-libs/src/libs/hydrateAndFilter.js]: I chose to still display this parameter to an eventual user of the shouldUpdate option that this parameter is available.

File: packages/cozy-konnector-libs/src/libs/linkBankOperations.js (hunk 4 above)
Comment (doubleface) [reply packages/cozy-konnector-libs/src/libs/linkBankOperations.js]: I don't know, I don't to use globals. I have no better argument

**Pull request merged**